### PR TITLE
wth - (SPARCDashboard) View Epic Queue Reconfigurations: Part 1

### PR DESCRIPTION
**Pull request by wtholt**
1. When a user double clicks on a row, they are taken to the
Dashboard::Protocol#show for the associated Protocol.
2. Added 3 more columns, 'Current Queue Date' (epic_queues.created_at),
'By', (first and last name attributes in epic_queues.identity...this
association was also missing), and
'Send' (Protocols#push_to_epic).
3. Changed title of tab to 'Current'.
[#141707817]

### CODE REVIEW

File: app/helpers/dashboard/epic_queues_helper.rb [hunk 1]
```diff
@@ -30,12 +30,25 @@ def format_pis(protocol)
     end
   end
 
+  def format_name(epic_queue)
+    identity = epic_queue.identity
+    "#{identity.try(:first_name)} #{identity.try(:last_name)}"
```
Comment (kayla-glick) [inline app/helpers/dashboard/epic_queues_helper.rb]: Identity already has #full_name which does this. So you could just return `epic_queue.identity.full_name`

File: app/views/dashboard/epic_queues/_epic_queue_list.html.haml [hunk 2]
```diff
@@ -31,9 +31,15 @@
               = t(:epic_queues)[:protocol]
             %th{data: { field: "pis", align: "left", sortable: 'true' } }
               = t(:epic_queues)[:PIs]
+            %th{data: { field: 'created_at', align: 'left', sortable: 'true' } }
+              Current Queue Date
```
Comment (kayla-glick) [inline app/views/dashboard/epic_queues/_epic_queue_list.html.haml]: Needs to be YMLized

File: app/views/dashboard/epic_queues/_epic_queue_list.html.haml [hunk 3]
```diff
@@ -31,9 +31,15 @@
...
+            %th{data: { field: 'name', align: 'left', sortable: 'true' } }
+              by
```
Comment (kayla-glick) [inline app/views/dashboard/epic_queues/_epic_queue_list.html.haml]: Ditto

File: app/views/dashboard/epic_queues/_epic_queue_list.html.haml [hunk 4]
```diff
@@ -31,9 +31,15 @@
...
             %th{data: { field: "date", align: "left", sortable: 'true', sorter: "dateSorter" } }
               = t(:epic_queues)[:date]
             %th{data: { field: "status", align: "left", sortable: 'true' } }
               = t(:epic_queues)[:status]
+            %th{data: { field: 'send', align: 'center', sortable: 'true' } }
+              Send
```
Comment (kayla-glick) [inline app/views/dashboard/epic_queues/_epic_queue_list.html.haml]: Ditto

File: app/views/protocols/push_to_epic.js.coffee [hunk 5]
```diff
@@ -0,0 +1 @@
+$('.epic-queue-table').bootstrapTable('refresh')
```
Comment (kyle-glick) [inline app/views/protocols/push_to_epic.js.coffee]: Missing copyright text

**Review by kayla-glick — CHANGES_REQUESTED** [review]

File: app/helpers/dashboard/epic_queues_helper.rb (hunk 1 above)
Comment (wtholt) [reply app/helpers/dashboard/epic_queues_helper.rb]: Oh yeah, nice catch. Forgot about that method.

**Review by kayla-glick — APPROVED** [review]

**Review by jleonardw9 — APPROVED** [review]

**Review by Stuart-Johnson — APPROVED** [review]

**Pull request merged**